tests/resources/helper_functions.py

```diff
@@ -217,11 +217,11 @@ def deploy_erc721_erc20(
     config: Config,
     erc721_publisher: Wallet,
     erc20_minter: Optional[Wallet] = None,
+    cap=Web3.toWei("0.5", "ether"),
```

Comment:
Cap argument added as it was accidentally removed here: https://github.com/oceanprotocol/ocean.py/commit/f28d07c0300f2f4810f193708531e78004eab0c1